mltsp/cfg.py

```diff
@@ -227,8 +227,9 @@
 # Specify number of time series to featurize as part of a "test run"
 TEST_N = 5
 
-# Specify default measurement error value (for measurements without errors)
-ERROR_VALUE = 1e-4
+# Specify default values for missing times/errors
+DEFAULT_MAX_TIME = 1.0
+DEFAULT_ERROR_VALUE = 1e-4
```

Comment:
I would like for it to, but currently a lot of the old TCP features will break if errors aren't passed in, or if they're taken to be zero. Even worse, we'd actually get different feature values if we changed this to 1e-1 or 1e-10; I'm just choosing 1e-4 to stay consistent with the old code.